src/es6Remx/useRemxValue.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import * as mobx from 'mobx';
+import * as Logger from './logger';
+import useForceUpdate from '../utils/useForceUpdate';
+
+const useRemxValue = (getter, ...params) => {
```

Comment:
I thaught we agreed to not introduce another hook but just modify useConnect hook so that it’s dependencies are passed as function params

Reply:
I think it's worth adding this. Having individual hooks returning single values is more intuitive in hook world I feel. I also didn't want to change any existing API.

Reply:
Look at the difference between UseConnectComponent and UseRemxComponent for example

Reply:
We can modify useConnect hook, so that it works just like useRemxValue allowing to return individual values. That shouldn't be a breaking change for existing API. I don't like copy-pasting useConnect with small tweaks that results in having 2 almost identical hooks in remx API

Reply:
I don't see a way to modify useConnect in a way that allows it to pass values while keeping the current API, and if it does, then connect will be the wrong name.

Reply:
Ok, I'll make a PR myself later, then we can discuss renaming useConnect or making an alias

Reply:
Check #59